### PR TITLE
Roi changes

### PR DESCRIPTION
**Pull request by manuarya27**
Creating PR for ROI demo changes.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,6 +18,8 @@ COPY src/compute_metrics_script.py compute_metrics_script.py
 COPY src/gen_data_and_simulate_drift.py gen_data_and_simulate_drift.py
 COPY src/fake_data_script.py fake_data_script.py
 COPY src/training_script.py training_script.py
+COPY src/roi.py roi.py
```
Comment (jananjoy) [inline Dockerfile:21]: refer the name of the file in the Roi pul342 development  branch "roi_backend.py"

File: src/roi.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+#  Author:  Jananjoy Rajkumar <jrajkumar@rocketscience.one>
+import numpy as np
```
Comment (jananjoy) [inline src/roi.py:2]: Update with the new file( roi_backend.py) in the branch "Roi pul342 development "

File: src/compute_metrics_script.py [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
-from pulsar_data_collection.data_capture import DataCapture, DatabaseLogin, DataWithPrediction
+from pulsar_data_collection.data_capture import DataCapture, DatabaseLogin
 from pulsar_metrics.analyzers.base import Analyzer
+from pulsar_metrics.metrics.base import CustomMetric 
+from roi import ReturnOfInvestment
 
+@CustomMetric
+def test_roi(current, reference, multiple=3, **kwargs):
+    roiInstance = ReturnOfInvestment()
+    roi_x = roiInstance.calculate_ROI(reference, current)
```
Comment (jananjoy) [inline src/compute_metrics_script.py:12]: change the function signature of the calculate_ROI() methods . Please refer the branch "Roi pul342 development "

File: README.md [hunk 4]
```diff
@@ -10,11 +10,17 @@ Thsi repo containing an example use-case showing how to leverage [pulsar-data-co
     ```bash
         export GRAFANA_USERNAME=admin;
         export GRAFANA_PASSWORD=pass123;
+        export DB_USER=admin;
+        export DB_PASSWORD=pass123;
```
Comment (rs-aben) [inline README.md:14]: Not sure having the password and username in clear is a good idea. better to use a key vault.

File: src/compute_metrics_script.py [hunk 5]
```diff
@@ -43,12 +60,65 @@
...
+        # @CustomMetric
+        # def test_roi(current, reference, multiple=3, **kwargs):
+        #     return multiple*np.max(current - reference)
+
+        # cus = test_roi(metric_name = 'test')
+        # cus.evaluate(current = current_data, reference = reference_data, threshold = 1, multiple=0.5)
+        # df_roi = cus.get_result()
+        # print("df_roi: ", df_roi)
+      
+
```
Comment (rs-aben) [inline src/compute_metrics_script.py:121]: In the new version of pulsar-metrics, you don't need to do it this way. We can add the custom metric to the analyzer directly. I invite to look at my notebook there.

File: src/fake_data_script.py [hunk 6]
```diff
@@ -25,37 +32,68 @@
 #     prediction = model.predict(test_data)
 
 # Below is the code for Pokemon model
-pokemon_test_data='pokemon.csv'
+pokemon_test_data='/app/datasets/bank_num.csv'
```
Comment (rs-aben) [inline src/fake_data_script.py:35]: Should change the name so it is not tied to the Pokemon dataset

**Review by rs-aben — APPROVED** [review]
There are some minor changes to have a better form of the code. If the code is running, it can stay like this for now.

**Pull request merged**